server/server.go

```diff
@@ -0,0 +1,37 @@
+package server
+
```

Comment:
Hint:
Create user.go file with `type User struct` with json tags and converter form `db.User` struct